doc/source/turn-off.rst

```diff
@@ -14,10 +14,14 @@ Tearing down your JupyterHub entails:
 3. Running a final check to make sure there aren't any lingering resources that haven't been deleted
    (e.g., storage volumes in some cloud providers)
 
-
 For all cloud providers
 -----------------------
 
+.. _delete-namespace:
+
+Delete the helm namespace
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Should we go ahead and be consistent with the caps on step 1 and 2.

Reply:
Hey - could you clarify what you mean? I'm not sure which caps you're talking about

Reply:
In the next few lines (which are not edited in the initial PR):

helm delete <your-helm-release-name> --purge

helm delete <YOUR-HELM-RELEASE-NAME> -- purge

kubectl delete namespace <YOUR-NAMESPACE>